components/Link.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import NextLink from 'next/link'
+
+export default function Link({ children, className, rel, ...restProps }) {
+    return (
+        <NextLink
+            rel={rel ? rel : "noreferrer"}
+            className={className ? className : "text-blue-600 underline decoration-dotted dark:text-blue-500 hover:underline hover:decoration-solid"}
+            {...restProps}
```

Comment:
rather than explicitly passing props into this component, I think it's a bit cleaner to just spread whichever props are not given default values down to the NextLink